src/resolvers/invitation.resolvers.ts

```diff
@@ -15,7 +15,7 @@ const invitationResolvers: IResolvers = {
   Mutation: {
     sendInvitation: async (
       _,
-      { invitees, orgToken,type}: { invitees: { email: string; role: string }[]; orgToken: string;type:any},
+      { invitees, orgToken,type}: { invitees: { email: string; role: string, name: string }[]; orgToken: string;type:any},
```

Comment:
Remive this name because inviting user is only done by providing email and role. think of the person who is emplemeting inviting user by excel file. that file has email and role ony that is why you don't need the name. if it is applicble on the figma ignore it,